.gitignore

```
@@ -7,3 +7,6 @@ out-js/
 node_modules/
 tmp/
 package-lock.json
+tutorial/out-js/
```

think this should be file names not a dir file
```suggestion
tutorial/out-js.*
```
https://github.com/AlexSmith/hoshie-lang/blob/62cd2fdbab0e51990087123ec0199180dd406670/src/hlcc/codeGen/js.ts#L18

---

It will ignore everything inside out-js,
The proposed changed would only ignore files inside tutorial that are named out-js with any file extension 
"out-js.*"